projecttacoma/deqm-test-server-frontend
# Evaluate measure request preview

## Summary
Rendering of the evaluate measure request preview. 
## New behavior
On the Evaluate Measure page for a valid Measure, a preview of the evaluate measure request is displayed under the DatePickers, Radio buttons, and auto-complete components. As the user selects dates, the reportType, Practitioner, and Patient (if applicable), the request preview changes to reflect the current selections.
## Code changes
- **pages/[resource]/[id]/evaluate.tsx**  renders request preview 
- **__tests__/pages/resource/id/evaluate.test.tsx**, unit tests added to file, some unit tests altered, some removed and added to MeasureDatePickers.test filed
- **__tests__/components/MeasureDatePickers.test.tsx**, added file and tests
- **components/MeasureDatePickers.tsx**, comment added
## Testing Guidance
- See the README.md for first time setup instructions.
#### Testing in browser: 
To start the frontend: `npm run dev` then navigate to http://localhost:3001 in a browser
- Click on “Measure” from the left-hand-side navigation menu. Click on any of the IDs. Click on the “Evaluate Measure” button near the top right of the main page body.
- Change the values of the DatePickers, Radio buttons, and AutoComplete components and see the "Request Preview" change to accurately reflect the current values of those components.
#### Unit testing: 
- Run the unit tests: `npm run test`
